CommunityToolkit.Tooling.SampleGen/ToolkitSampleMetadataGenerator.Sample.cs

```diff
@@ -23,16 +22,14 @@ public void Initialize(IncrementalGeneratorInitializationContext context)
     {
         var symbolsInExecutingAssembly = context.SyntaxProvider
             .CreateSyntaxProvider(
-                static (s, _) => s is ClassDeclarationSyntax c && c.AttributeLists.Count > 0,
+                static (s, _) => s is ClassDeclarationSyntax c && c.AttributeLists.Count > 0 || s is MethodDeclarationSyntax m && m.AttributeLists.Count > 0,
                 static (ctx, _) => ctx.SemanticModel.GetDeclaredSymbol(ctx.Node))
```

Comment:
You shouldn't pass symbols across compilation steps.
Though I guess you were doing this already.

Reply:
Originally copied from the MVVM toolkit 😅
We can / should look at performance improvements, but in a dedicated PR.

Reply:
Must've been an old branch, because I stopped doing that months ago 😛